doc/plotting/index.rst

```diff
@@ -231,6 +231,32 @@ Different display modes
 
 ================= =========================================================
 
+Available Colormaps
+========================
+
+Nilearn's plotting library ships with a set of extra colormaps, as seen in the
+image below::
```

Comment:
This should be put in a example, ie a Python file in examples/plottting/plot_colormaps.py. These get executed by our build system when you build the docs.